packages/fastify/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@byu-oit/fastify-jwt",
+  "version": "0.1.0-beta.6",
+  "description": "A fastify plugin for verifying callers JWTs with the BYU JWT package",
```

Comment:
```suggestion
  "description": "A Fastify plugin for verifying callers' JWTs with the BYU JWT package",
```

Reply:
The README probably needs the same update.

Reply:
✅